Dependencies outdated?

Since Blessed isn't been maintain can this be used with neo-blessed?

> are there specific deps that need to be updated?

`marked-terminal` seems out of date, I've been running in to the following using the `markdown` module:

```
[ERROR] 09:12:10 TypeError: this.renderer.checkbox is not a function                                                                                                       │
Please report this to https://github.com/markedjs/marked
```

Searching the marked issues, it seems this is actually a `marked-terminal` bug (which is apparently fixed): https://github.com/markedjs/marked/issues/1323#issuecomment-415813602

--- a/examples/marked-terminal.js
+++ b/examples/marked-terminal.js
@@ -1,0 +1,8 @@
+const blessed = require("blessed")
+const contrib = require("../")
+
+const screen = blessed.screen()
+const markdown = contrib.markdown()
+screen.append(markdown)
+markdown.setMarkdown("- [x] Checkbox")
+screen.render()

--- a/package.json
+++ b/package.json
@@ -33,7 +33,7 @@
     "lodash": "~>=4.17.11",
     "map-canvas": ">=0.1.5",
     "marked": "^0.7.0",
-    "marked-terminal": "^1.5.0",
+    "marked-terminal": "^4.0.0",
     "memory-streams": "^0.1.0",
     "memorystream": "^0.3.1",
     "picture-tuber": "^1.0.1",

